extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -533,6 +533,24 @@ public void setRoleClaimSeparator(String roleClaimSeparator) {
         @ConfigItem
         public Optional<String> cookiePath = Optional.empty();
 
+        /**
+         * If this property is set to 'true' then a normal 302 redirect response will be
```

Comment:
@boosey I propose to tweak this doc section a bit. 
```
If this property is set to 'true' then a normal 302 redirect response will be returned if the request was initiated via XMLHttpRequest and the current user needs to be (re)authenticated which may not be desirable for Single Page Applications since XMLHttpRequest automatically following the redirect may not work given that OIDC authorization endpoints typically do not support CORS.
If this property is set to `false` then a status code of '499' will be returned to allow the client to handle the redirect manually
```
Something along these lines ( please tweak as needed to make it a better English) as we need to be very clear it is all about XHR and when exactly the users may want to do it.

Reply:
Done.